### DISCUSSION TITLE
Can we get multiple Start Menu icons

### DISCUSSION THREAD

**Discussion by jamaddox** [Rem0o/FanControl.Releases#2355] [General]
Unfortunately my start menu is off white and the icon for the start menu is all white.  So on my start menu I just have the title and that is it.  And also for the main taskbar the icon is white as well.  Note the icon in the notification are is black

**Comment by jamaddox** [comment]
I actually 'fixed' this. I did a personalize in the settings, and used the dark theme for windows mode but the light theme for the default app mode.  I see the fan on the taskbar and the start menu, But since the notification also has a black bar then the icon in the notification area just blends in

**Reply by Rem0o** [reply]
Use tray icon temperature and pick whatever color you like: 
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/7b7c56f8-032f-42d5-b882-8ea79bb6339b)
